editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/wizards/FuseIntegrationProjectCreatorRunnable.java

```diff
@@ -112,6 +118,9 @@ public void run(IProgressMonitor monitor) throws InvocationTargetException, Inte
 		// refresh
 		try {
 			c.getProject().refreshLocal(IProject.DEPTH_INFINITE, subMonitor.newChild(1));
+			// update the manifest to reflect project name as Bundle-SymbolicName
+			updateBundleSymbolicName(c.getProject(), subMonitor.newChild(1));
+			c.getProject().refreshLocal(IProject.DEPTH_INFINITE, subMonitor.newChild(1));

```

Comment:
as 2 "newChild are called, the line SubMonitor subMonitor = SubMonitor.convert(monitor, Messages.FuseIntegrationProjectCreatorRunnable_CreatingTheProjectMonitorMessage, 6); should be modified to 8